output/client_test.go

```diff
@@ -28,11 +28,20 @@ func TestRequestHeaders(test *testing.T) {
 	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		require.Equal(test, "gzip", r.Header.Get("Content-Encoding"))
 		require.Equal(test, "application/json", r.Header.Get("Content-Type"))
+		require.Equal(test, "header_value_1", r.Header.Get("X-Custom-Header1"))
+		require.Equal(test, "header_value_2", r.Header.Get("X-Custom-Header2"))
 		w.WriteHeader(http.StatusOK)
 	}))
 	defer testServer.Close()
 
-	logzioClient := LogzioTestClient(testServer.URL)
+	headers := map[string]string{
```

Comment:
I think we need coverage for more cases like:
1. Test with No Headers
2. Test with Duplicate Headers
3. Test with Malformed Header String (missing `:`)
4. Test with Mixed Valid and Invalid Headers

Reply:
I added no header test.
About the duplication scenario, a headers map will only store the last value for a given header key if there are duplicates. Go maps have unique keys, so if a user sets duplicate headers like X-Custom-Header1: header_value_1, X-Custom-Header1: header_value_2, only header_value_2 will be retained. The logging I added to 'initConfigParams' func in 'out_logzi.go' will notify the user of any duplicate keys being overwritten.